woocommerce/woocommerce-gutenberg-products-block
# Add storybook entry for CheckboxList component

## What
Thanks to #11524 storybook is back up and running. This PR adds a storybook entry for the newly exported CheckboxList component


## Why

So that external developers have good documentation for exported components

## Testing Instructions



1. Build the project
2. Run storybook - `npm run storybook`
3. Check the entry for the CheckboxList component. It should be under the section "External Components". Check the docs appear as they should and the component reflects changes made to props



## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:
